Doc/tutorial/modules.rst

```diff
@@ -585,8 +585,7 @@ as the main module of a Python application must always use absolute imports.
 Packages in Multiple Directories
 --------------------------------
 
-Packages support one more special attribute, :attr:`__path__`.  This is
-initialized to be a list containing the name of the directory holding the
+All packages have a special attribute :attr:`__path__`.  This is initialized to be a list containing the name of the directory holding the
```

Comment:
```suggestion
All packages have a special attribute :attr:`__path__`. This is
initialized to be a list containing the name of the directory holding the
```

Minimize diffs and avoid unnecessary overflow

Reply:
Fixed.